lint.go

```diff
@@ -7,20 +7,50 @@ import (
 	"github.com/qri-io/jsonschema"
 	"io"
 	"net/http"
+	"os"
 )
 
-func Lint(dataContractLocation string, schemaUrl string, target io.Writer) error {
+func GetSchema(location string) (*jsonschema.Schema, error) {
```

Comment:
The function is only used here and therefore should be private (start with a lower case character, that's a Go convention). `getSchema`

It's extra important here because the term "schema" is ambiguous.